### DISCUSSION TITLE
Display the composer? Separate columns composer/artist instead of "peoples"

### DISCUSSION THREAD

**Discussion by Twix53791** [quodlibet/quodlibet#3824] [Q&A] [answered]
Hello,

I think than aggregate in a field the composer and the interpreter (artist) is confusing : it leads to a mess (as it can be many artists...) ! You have the option to just uncheck "the artist contains all the people" to display only the artist... but so, where is the composer (which is for me the most important thing to order my library!) ? I tried to add a custom field : ~composer. But it didn't work... Maybe I do it wrong? I sure my flac files contain both the field artist and composer, so the issue don't follow the files...

Thank you for your consideration,
A nice day

**Comment by Ironbalden** [comment, marked as answer]
It's just composer, ~ is only for internal tags https://quodlibet.readthedocs.io/en/latest/guide/tags/internal_tags.html#internaltags

**Reply by Twix53791** [reply]
Ah! Ok. Perfect. Thanks for the wiki pages, I understand better now.

Thank you
